trailblazer/services/job_service/utils.py

```diff
@@ -1,13 +1,45 @@
-from trailblazer.dto.failed_jobs_response import FailedJobsResponse
-from trailblazer.dto.job_response import JobResponse
+from pathlib import Path
+from trailblazer.constants import FileFormat, SlurmJobStatus, TrailblazerStatus
+from trailblazer.io.controller import ReadFile
 from trailblazer.store.models import Job
 
 
-def create_job_response(job: Job) -> JobResponse:
-    return JobResponse(
-        slurm_id=job.slurm_id, analysis_id=job.analysis_id, status=job.status, id=job.id
+def get_slurm_job_ids(job_id_file: str) -> list[int]:
+    job_id_file_path = Path(job_id_file)
+    content: dict = ReadFile.get_content_from_file(
+        file_format=FileFormat.YAML, file_path=job_id_file_path
     )
+    job_ids: list[int] = []
+    for row in content.values():
+        [job_ids.append(job_id) for job_id in row]
+    return job_ids
 
 
-def create_failed_jobs_response(failed_job_statistics: list[dict]) -> FailedJobsResponse:
-    return FailedJobsResponse(jobs=failed_job_statistics)
+def get_status(jobs: list[Job]) -> TrailblazerStatus:
+    is_running: bool = has_running_jobs(jobs)
+    is_failed: bool = has_failed_jobs(jobs)
+    if is_failed and is_running:
+        return TrailblazerStatus.ERROR
+    if is_failed:
+        return TrailblazerStatus.FAILED
+    if is_running:
+        return TrailblazerStatus.RUNNING
+    return TrailblazerStatus.CANCELLED
```

Comment:
Is the old function not removed in this PR? I think there might be one piece of logic missing here:
```
    if single_analysis_status := _get_analysis_single_status(jobs_status_distribution):
        return single_analysis_status
```

Reply:
No, I'm leaving removing the deprecated code for a separate PR. It requires untangling a lot of other code.

Added the old logic here https://github.com/Clinical-Genomics/trailblazer/pull/423/commits/cf838778fcad580369e55230caf429304031bdf8.